scripts/superdesk-authoring/metadata/metadata.js

```diff
@@ -457,8 +457,10 @@ function MetadataListEditingDirective(metadata, $filter) {
                         scope.activeTree = _.without(scope.activeTree, term);
                     }
 
-                    scope.postprocessing();
-                    scope.change({item: scope.item});
+                    $timeout(function() {
+                        scope.postprocessing();
+                        scope.change({item: scope.item});
+                    }, 50);
```

Comment:
probably there is no need to trigger full digest, so it would be better to use `50, false);` and call `scope.applyAsync` in timeout